src/currentBrowser.js

```diff
@@ -0,0 +1,15 @@
+import { detect as detectBrowser } from 'detect-browser'
+import capitalize from 'lodash/capitalize'
+import platforms from 'platforms'
```

Comment:
Nit: The platforms file is not advertised as a "supported platforms" file. It is a bit dangerous since someone could add a platform there and it would automatically be understood a "supported" in the currentBrowser.js context. Ideally I think a `supported: true` attribute on the platforms side could be used. Incidentally, do you think the content of this file could be merged in platforms.js  with a name like `crossbrowser.js` ?

Reply:
In my mind `currentBrowser`'s role is to give us informations about the current browser (do we have an extension for it? what's its name?); and `platforms` lists supported platforms with links to the according stores (currently we have firefox and chrome, but we may have android and ios in the future). So I don't think these files should be merged, as it doesn't have the same goal. But I think renaming `platforms` to `supportedPlatforms` would be a good thing.

Reply:
We have an extension with helpers for the platform here but I am not sure it fits the bill here: https://github.com/cozy/cozy-libs/tree/master/packages/cozy-device-helper

Reply:
I considered it, but here we want to have the browser's name (firefox, chrome, safari...). Maybe we can think about adding this information to cozy-device-helper though.

Reply:
See https://github.com/cozy/cozy-libs/issues/901